### PR TITLE
initial spec for kamon reporter data

### PR DESCRIPTION
**Pull request by breedx-nr**

### CODE REVIEW

File: kamon/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Kamon 
+
+New Relic [provides two reporters for Kamon telemetry](https://github.com/newrelic/kamon-newrelic-reporter):
```
Comment (jkwatson) [inline kamon/README.md]: also add in a link to kamon itself, please.

Comment (breedx-nr) [reply kamon/README.md]: good call!

File: kamon/README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+* `traceId`: `kamonSpan.trace.id.string`
+* `parentId` `kamonSpan.parentId`
+* `name`: `kamonSpan.operationName`
+* `timestamp`: Microseconds->millisecond conversion of `kamonSpan.from`
+
+## Span specific attributes:
+
+Each span has the following attributes:
+
+* `span.kind`: `kamonSpan.kind.toString`
```
Comment (jkwatson) [inline kamon/README.md]: this doesn't look right

File: kamon/README.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+All spans will have the following common attributes on them:
+* `instrumentation.source`: "kamon-agent"
+* `service.name`: value comes from the config at `kamon.environment.service`
+
+## Span field data
+
+Other span field data maps over pretty directly.  The field on the left is the new relic
+span field name, and the right is where it is sourced:
+
+* `spanId`: `kamonSpan.id.string`
```
Comment (jkwatson) [inline kamon/README.md]: maybe write this as just <kamon provided span id>? With the backticks, it makes it look like a literal

**Pull request merged**